S23/thofstrand/lowerCase.py

```diff
@@ -0,0 +1,76 @@
+import pandas as pd
+
+
+
+def lowerCase(word, lang):
+    word = word.strip()
+    langStart = lang[:2]
```

Comment:
We can add a condition if the user is giving false/errorful language as input.

Reply:
Good thought! I am adding this now!

Reply:
This code change is fine.